envoy/stream_info/filter_state.h

```diff
@@ -66,6 +96,16 @@ class FilterState {
     virtual absl::optional<std::string> serializeAsString() const { return absl::nullopt; }
   };
 
+  struct FilterObject {
+    std::shared_ptr<Object> data_;
+    StateType state_type_;
```

Comment:
```suggestion
    StateType state_type_{};
```
default initialize this?

Reply:
Done.